packages/p2p/src/components/dp2p-blocked/dp2p-blocked-checklist.jsx

```diff
@@ -9,7 +9,7 @@ const Dp2pBlockedChecklist = () => {
     const { general_store } = useStores();
     const { history } = general_store.props;
 
-    if (general_store.is_high_risk_fully_authed_without_fa && !general_store.is_p2p_blocked_for_pa) {
+    if (general_store.is_high_risk) {
```

Comment:
Why're we removing `is_high_risk_fully_authed_without_fa`? It's not the same as `is_high_risk`, right?

Reply:
I removed it because when the user is high risk, for all cases, it should display the checklist